test/CMakeLists.txt

```diff
@@ -49,7 +49,7 @@ add_executable(io-test
 add_executable(weak-test ${CMAKE_CURRENT_LIST_DIR}/weak.cpp)
 
 target_link_libraries(io-test PRIVATE
-    Mapbox::Base::io
+    Mapbox::Base::Static::io
```

Comment:
I don't think we need a new namespace for static libraries.

Reply:
We can't make it dependant on `mapbox-base` though as it is an interface library.